Set up the project by adding linters and main files

DavidCazco36 opened the pull request:
**Main changes in this branch:**

- Installed the linters for html and css.
- Added a gitignore file to ignore the node_modules created by the npm package manager.
- Added three main files: index.html, styles.css and test.md

DavidCazco36 committed:
Set Up GitHub Actions

DavidCazco36 committed:
Read Me File Update

add the relevant information for this project

DavidCazco36 committed:
add html file

DavidCazco36 committed:
add a css file

DavidCazco36 committed:
add the test.md file to gitignore

DavidCazco36 committed:
Linter SetUp for html and css

DavidCazco36 committed:
Unnecessary information was deleted

final update of my readme file.

fherrerao commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,44 @@
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.1.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.rubocop.yml
+      - name: Rubocop Report
+        run: rubocop --color
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
- [ ] Good job adding linters 👍🏼, but you added the wrong linters. Kindly, review [this documentation](https://github.com/microverseinc/linters-config/tree/master/html-css) and update the `linters.yml` file. 🦺

DavidCazco36 replied:
@fherrerao I did add the linters.yml file that you mention. And I followed all the steps in [this documentation](https://github.com/microverseinc/linters-config/tree/master/html-css), so please let me know what else can I do to install the required linters.

DavidCazco36 replied:
@fherrerao I used this file [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) that [this documentation](https://github.com/microverseinc/linters-config/tree/master/html-css) tell me to use :( I need help.

fherrerao reviewed (changes requested):
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#
Hi @DavidCazco36 💯, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![So Close Snl GIF by Saturday Night Live](https://user-images.githubusercontent.com/91301423/197784121-950974af-5415-4bf0-871b-ce226a51ace4.gif)


## Highlights

✔️  Good use of GitHub flow.
✔️  Good README
✔️  Good Commit Messages


## Required Changes ♻️

- [ ] Kindly, the pull request should have a descriptive title 🦅 telling you what changes are inside. It could be better if you update it 🪜.
- [ ] Kindly, every pull request should have a summary ✌🏼 describing introduced changes in more detail. Please, update your summary with detailed information. ⚡

![image](https://user-images.githubusercontent.com/91301423/204688726-d5fde1b2-bdaa-47fa-8b69-91456433a49b.png)

- [ ] Please, also _check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DavidCazco36 committed:
Clean all and restart

DavidCazco36 committed:
Test

DavidCazco36 committed:
Create Readme.md

DavidCazco36 committed:
Add a git ignore file

DavidCazco36 committed:
Add linters.yml file

DavidCazco36 committed:
Add HTML and CSS linters files

DavidCazco36 committed:
Add HTML file

DavidCazco36 committed:
Add CSS file

DavidCazco36 committed:
Corrected CSS format

DavidCazco36 commented:
**Main changes in this branch:**
- Installed the linters for html and css.
- Added a gitignore file to ignore the node_modules created by the npm package manager.
- Added three main files: index.html, styles.css and test.md

brainconnect93 reviewed (changes requested):
### Hi @DavidCazco36 , 👋 

# STATUS: Changes Required :recycle:

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 


<img src="https://media.tenor.com/M5SonVIGcOMAAAAj/rascal-almost-there.gif">


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 
- Files properly ignored  🥇 

### Changes Required

- [ ] As suggested by the the previous reviewer, i noticed you only make changes to the pull request title, while the pull request comments where left unrectified. I would suggest you give you pull request a well descriptive comment in order for people to know what you have done in the project. Kindly refer to the Link provided by the previous reviewer to learn more about the best way to write pull request comments.
_thanks_

- [ ] You can simply edit the pull request comments by clicking on the three dots `. . .` , you will see the edit part, kindly click on that and edit your PR comment to be well detailed

<img src="https://user-images.githubusercontent.com/91301423/204688726-d5fde1b2-bdaa-47fa-8b69-91456433a49b.png">


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rskinnerc reviewed (approved):
Hi @DavidCazco36 👋👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![giphy](https://user-images.githubusercontent.com/26911718/204933283-9c828b71-b7ae-4b8b-a436-ad8df65d45bc.gif)

## Highlights
- You implemented the requested changes!! Now your PR summary looks more professional and descriptive. Making this a habit would make you a better dev. Congrats! 🎉🎈🎊🔥

## Optional suggestions
_Nothing here this time!_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
